packages/ra-core/src/controller/input/ReferenceArrayInputController.spec.tsx

```diff
@@ -198,34 +205,15 @@ describe('<ReferenceArrayInputController />', () => {
                 {children}
             </ReferenceArrayInputController>
         );
-        assert.equal(children.mock.calls[0][0].warning, undefined);
-    });
-
-    it('should pass onChange down to child component', () => {
-        const onChange = jest.fn();
-        const children = jest.fn();
-        shallow(
-            <ReferenceArrayInputController
-                {...defaultProps}
-                allowEmpty
-                onChange={onChange}
-            >
-                {children}
-            </ReferenceArrayInputController>
-        );
-        assert.equal(children.mock.calls[0][0].onChange, onChange);
+        expect(children.mock.calls[0][0].warning).toEqual(undefined);
     });
 
-    it('should call crudGetMatching on mount with default fetch values', () => {
+    it.skip('should call crudGetMatching on mount with default fetch values', () => {
```

Comment:
skip detected, here and down under

Reply:
Yes, because this PR is WIP ;) It event has a `Fix tests` in its description!